src/spanningtrees/boruvka.jl

```diff
@@ -0,0 +1,73 @@
+"""
+boruvka_mst(g, distmx = weights(g); minimize = true)
+Return a tuple `(mst, weights)` where `mst` is a vector of edges representing the
+optimum (minimum, by default) spanning tree of a connected, undirected graph
+`g` with optional matrix `distmx` that provides distinct edge weights, and
+`weights` is the sum of all the edges in the solution by using
+[Boruvka's algorithm](https://en.wikipedia.org/wiki/Bor%C5%AFvka%27s_algorithm).
+The algorithm correctly gives a minimum/maximum spanning tree if all edges have
+different weights.
+### Optional Arguments
+- `minimize=true`: if set to `false`, calculate the maximum spanning tree.
+"""
+function boruvka_mst end 
+
+# see https://github.com/mauro3/SimpleTraits.jl/issues/47#issuecomment-327880153 for syntax
```

Comment:
you can remove this line I think, the comment is already there in many other places

Reply:
removed it! thanks